ui/src/app/pages/analysis/runtime-panel.tsx

```diff
@@ -245,8 +246,13 @@ export const ConfirmDeleteUnattachedPD = ({onConfirm, onCancel}) => {
 
     <div style={styles.confirmWarning}>
       <h3 style={{...styles.baseHeader, ...styles.bold, gridColumn: 1, gridRow: 1}}>
-        <label ><input type= 'radio' checked={deleting === true} onChange={() => setDeleting(true)}/>
-          Delete persistent disk</label></h3>
+        <div data-test-id='Delete unattached pd' style={{display: 'inline-block', marginRight: '0.5rem'}}>
```

Comment:
nit: following the pattern for other data-test-ids, I would just use hyphenated, no spaces: e.g. `delete-unattached-pd`